sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -203,6 +205,12 @@ private boolean isAllowedApiRequest(HttpServletRequest request) {
 			}else if (url.matches(props.getString(CANVAS_API_GETCOURSE_INFO))) {
 				M_log.debug("The canvas api request for getting course info");
 				return true;
+			}else if (url.matches(props.getString(CANVAS_API_GETSECTION_INFO))){
+				M_log.debug("The canvas api request for getting section info");
+				return true;
+			}else if (url.matches(props.getString(CANVAS_API_GETSECTION_PER_COURSE))) {
+				M_log.debug("The canvas api request for getting section info for a given course");
+				return true;
```

Comment:
Could an enumerator type for the constants being used and converting this section into a case statement be possible? Would it be more efficient?